APITestingFramework/StepDefinitions/User/Put.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Text.Json;
+using Core;
+using Features.GeneralSteps;
+using Models;
+using RestSharp;
+using TechTalk.SpecFlow;
+
+namespace Features.User.Put
+{
+    [Binding]
+    [Scope(Feature = "User Update")]
+    public class PutStepDefinitions : CommonSteps
+    {
+        private readonly ScenarioContext _scenarioContext;
+        private readonly RestHelper client = new RestHelper("https://todo.ly/api");
+
+        public PutStepDefinitions(ScenarioContext scenarioContext) : base(scenarioContext)
+        {
+            _scenarioContext = scenarioContext;
+        }
+
+        [When(@"the user submits a PUT request to the API endpoint with a valid JSON or XML payload")]
+        public void WhentheusersubmitsaPUTrequesttotheAPIendpointwithavalidJSONorXMLpayload()
+        {
+            var url = "user/0.json";
```

Comment:
the ID should be get from precondition

Reply:
Actually the ID is always gonna be "0". The API doesn't use it, it just modifies/deletes the current authenticated user.